kunlar/12-kun-Regular-ifodalar/12-kun-regular-ifodalar.md

```diff
@@ -0,0 +1,412 @@
+        print(qator)
+```
+
+Konsolda
+```commandline
+Jumladan, “Hokimiyat yoki mansab vakolatini suiiste’mol qilish”, “Hokimiyat yoki mansab vakolati doirasidan chetga chiqish”, “Hokimiyat harakatsizligi”, “Mansab soxtakorligi” va “Hokimiyatni suiiste’mol qilish, hokimiyat vakolatidan tashqariga chiqish yoki hokimiyat harakatsizligi” jinoyatlar agar “g‘arazgo‘ylik” niyatida sodir etilsa, korrupsiyaviy jinoyat hisoblanishi belgilanib, mazkur harakatlar uchun og‘irroq jazolar belgilangan.
+```
+
+## `.` Nuqta 
+> `.`  period, full stop (ingliz)
+
+Nuqta belgisi ixtiyoriy xarf bo'lishi mumkin degan ma'noni bildiradi. Masalan,  
+boshlanishi `y` va tugashi `i` belgisi bilan tugaydigan qatorlarni konsolga chiqarish
+kerak bo'lsin. Unda biz quyidagicha yozamiz `y..i`, yani ikki nuqta o'rniga ixtiyoriy harflar bo'lishi
+mumkinligini bildiradi. Buni quyidagi programmada qo'llasak,
+
+```python
+#  nuqtalar bilan ishlash, nuqtalar ixtiyoriy belgiga ruhsat borligini bildiradi
+fayl = open("regifoda.txt", 'r')
+for qator in fayl.readlines():
+    qator = qator.rstrip()
+    if re.search("y..i", qator):
+        print(qator)
+```
+
+Konsolda
+```commandline
+Jumladan, “Hokimiyat yoki mansab vakolatini suiiste’mol qilish”, “Hokimiyat yoki mansab vakolati doirasidan chetga chiqish”, “Hokimiyat harakatsizligi”, “Mansab soxtakorligi” va “Hokimiyatni suiiste’mol qilish, hokimiyat vakolatidan tashqariga chiqish yoki hokimiyat harakatsizligi” jinoyatlar agar “g‘arazgo‘ylik” niyatida sodir etilsa, korrupsiyaviy jinoyat hisoblanishi belgilanib, mazkur harakatlar uchun og‘irroq jazolar belgilangan.
+Shuningdek, o‘zlashtirish yoki rastrata yo‘li bilan firibgarlik qilish, soliq va boshqa majburiy to‘lovlarni to‘lashdan bosh tortish uchun hukm qilingan, biroq yetkazilgan zararni to‘liq qoplamagan shaxslarga va poraxo‘rlik bilan bog‘liq jinoyatni sodir etgan shaxslarga nisbatan jazodan muddatidan ilgari shartli ravishda ozod qilish yoki jazoni yengilrog‘i bilan almashtirish tatbiq qilinmasligi nazarda tutilgan.
+Received: from eyewitness.mr.itd.umich.edu (eyewitness.mr.itd.umich.edu [141.211.93.142])
+	by mission.mail.umich.edu () with ESMTP id m04JoHJi019755;
+	BY eyewitness.mr.itd.umich.edu ID 477E8DF2.67B91.5278 ;
+Subject: [sakai] svn commit: r39765 - in gradebook/trunk/app: business/src/java/org/sakaiproject/tool/gradebook/business business/src/java/org/sakaiproject/tool/gradebook/business/impl ui ui/src/java/org/sakaiproject/tool/gradebook/ui/helpers/beans ui/src/java/org/sakaiproject/tool/gradebook/ui/helpers/entity ui/src/java/org/sakaiproject/tool/gradebook/ui/helpers/params ui/src/java/org/sakaiproject/tool/gradebook/ui/helpers/producers ui/src/webapp/WEB-INF ui/src/webapp/WEB-INF/bundle ui/src/webapp/content/templates
+Received: from eyewitness.mr.itd.umich.edu (eyewitness.mr.itd.umich.edu [141.211.93.142])
+	BY eyewitness.mr.itd.umich.edu ID 477D8300.AC098.32562 ;
+```
+`y..i` ga o'shashso'zlar yuqoridagi qatorlarda quyidagicha kelgan:
```

Comment:
o'shashso'zlar - o'xshash so'zlar bo'lish kerak edi.

Reply:
To'g'irlandi. Rahmat.